### DISCUSSION TITLE
Sublime Text syntax highlighter

### DISCUSSION THREAD

**Discussion by patopesto** [charmbracelet/vhs#575] [Show and tell]
Hello,
I made a `.tape` files syntax highlighter for Sublime Text.
It's available [here](https://github.com/patopesto/Sublime-Text-VHS-Syntax) or through Package Control.
Would love to add it to the ReadMe section if that's okay.

PS: Wasn't sure if creating an issue for this was the right thing to do, so feel free to close it.

**Comment by caarlos0** [comment]
Hey @patopesto!

That's pretty great!

We don't have other editor plugins listed in the readme, but I suppose people will find it pretty easily either in the package manager, your repository, or, now, in through this issue.

In an effort to keep the backlog more or less organized, I'll convert this to a discussion.

Cheers!
